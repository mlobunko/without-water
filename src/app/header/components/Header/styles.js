import styled from 'styled-components';
import { NavbarBrand, NavLink } from 'reactstrap';
import imageSieve from '../../images/sieve.svg';

export const StyledNavbarBrand = styled(NavbarBrand)`
  background: url(${imageSieve}) no-repeat center center;
  background-size: contain;
  height: 2.5rem;
  width: 2.5rem;
`;

export const StyledNavLink = styled(NavLink)`
  &:hover {
    cursor: pointer;
  }
`;
