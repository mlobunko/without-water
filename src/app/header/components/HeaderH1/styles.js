import styled from 'styled-components';

export const StyledHeaderH1 = styled.h1`
  text-align: center;
  margin-top: 1rem;
  margin-bottom: 3rem;
`;

StyledHeaderH1.displayName = 'StyledHeaderH1';
