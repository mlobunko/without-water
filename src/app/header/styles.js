import styled from 'styled-components';
import { NavbarBrand, NavLink } from 'reactstrap';
import imageSieve from './images/sieve.svg';

export const StyledNavbarBrand = styled(NavbarBrand)`
  background: url(${imageSieve}) no-repeat center center;
  background-size: contain;
  height: 2.5rem;
  width: 2.5rem;
`;

export const StyledNavLink = styled(NavLink)`
  &:hover {
    cursor: pointer;
  }
`;

export const StyledHeaderH1 = styled.h1`
  text-align: center;
  margin-top: 1rem;
  margin-bottom: 3rem;
`;
