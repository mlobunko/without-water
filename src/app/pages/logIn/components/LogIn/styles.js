import styled from 'styled-components';

export const StyledButton = styled.div`
  text-align: center;
`;
StyledButton.displayName = 'StyledButton';

export const StyledForm = styled.form`
  margin: 0 auto;
  width: 90%;
  max-width: 400px;
`;

StyledForm.displayName = 'StyledForm';
