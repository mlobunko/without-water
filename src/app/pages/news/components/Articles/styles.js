import styled from 'styled-components';
import { Jumbotron } from 'reactstrap';

export const StyledArticlesSection = styled.div`
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: flex-start;
  flex-grow: 1;
`;
StyledArticlesSection.displayName = 'StyledArticlesSection';

export const StyledCounterNews = styled.div`
  text-align: center;
`;
StyledCounterNews.displayName = 'StyledCounterNews';

export const StyledJumbotron = styled(Jumbotron)`
  max-width: 700px;
  && {
    background-color: rgb(250, 250, 250);
  }
`;
StyledJumbotron.displayName = 'StyledJumbotron';

export const StyledHeader = styled.h3`
  margin-bottom: 2rem;
`;
StyledHeader.displayName = 'StyledHeader';
