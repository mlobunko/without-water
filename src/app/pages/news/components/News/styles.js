import styled from 'styled-components';

export const StyledNewsComponent = styled.div`
  flex-grow: 1;
  display: flex;
  flex-direction: column;
`;
StyledNewsComponent.displayName = 'StyledNewsComponent';
