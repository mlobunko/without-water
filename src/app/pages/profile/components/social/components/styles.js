import styled from 'styled-components';

export const StyledHeader = styled.h5`
  margin-top: 4rem;
  margin-bottom: 1rem;
`;

export const StyledSocialComponent = styled.div`
  width: 100%;
  display: flex;
  font-size: 2rem;
  justify-content: space-evenly;
`;
