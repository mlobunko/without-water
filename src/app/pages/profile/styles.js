import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
import styled from 'styled-components';

import imageUser from './images/imageUser.svg';

export const StyledFontAwesomeIcon = styled(FontAwesomeIcon)`
  color: rgb(52, 58, 64);
`;

StyledFontAwesomeIcon.displayName = 'StyledFontAwesomeIcon';

export const StyledProfileComponent = styled.div`
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  margin: 2rem auto;
  width: 95vw;
  max-width: 500px;
`;

StyledProfileComponent.displayName = 'StyledProfileComponent';

export const StyledSocialFontAwesomeIcon = styled(FontAwesomeIcon)`
  && {
    color: rgb(52, 58, 64);
    &:hover {
      color: rgb(161, 164, 165);
    }
    &:active {
      color: rgb(161, 164, 165);
    }
  }
`;

StyledSocialFontAwesomeIcon.displayName = 'StyledSocialComponent';

export const StyledUserImage = styled.div`
  background: url(${imageUser}) no-repeat center center;
  background-size: contain;
  height: 7rem;
  width: 100%;
  margin-bottom: 1rem;
`;

StyledUserImage.displayName = 'StyledUserImage';
