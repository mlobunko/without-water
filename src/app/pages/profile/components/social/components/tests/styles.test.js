import React from 'react';
import { shallow } from 'enzyme';
import 'jest-styled-components';

import { StyledHeader, StyledSocialComponent } from '../styles';

describe('StyledHeader', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<StyledHeader />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe('StyledSocialComponent', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<StyledSocialComponent />);
    expect(wrapper).toMatchSnapshot();
  });
});
