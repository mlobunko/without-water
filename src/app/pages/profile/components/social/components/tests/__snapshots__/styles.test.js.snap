// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledHeader renders correctly 1`] = `
.c0 {
  margin-top: 4rem;
  margin-bottom: 1rem;
}

<h5
  className="c0"
/>
`;

exports[`StyledSocialComponent renders correctly 1`] = `
.c0 {
  width: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  font-size: 2rem;
  -webkit-box-pack: space-evenly;
  -webkit-justify-content: space-evenly;
  -ms-flex-pack: space-evenly;
  justify-content: space-evenly;
}

<div
  className="c0"
/>
`;
